frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -43,7 +43,7 @@ li.c-item-wrapper(data-test='proposalItem')
         banner-scoped(ref='voteMsg' data-test='voteMsg')
         p.c-sendLink(v-if='invitationLink' data-test='sendLink')
           i18n(
-            :args='{ user: proposal.data.proposalData.memberName}'
+            :args='{ user: proposal.data.proposalData.memberID}'
```

Comment:
`memberID` is a variable name that is used to show the member's identity contractID.

But here we want a either a username or a display name.